tests/test_userhistory.py

```diff
@@ -1,33 +1,44 @@
-""" Tests for user history class """
+''' Tests for user history class '''
 import __init__
 from userhistory import UserHistory
 
 def test_userhistory_init():
-    """Tests user history init"""
-    # specifying a short date range prevents unnecessarily pulling down
-    # a very large number of revisions - saves us time on testing
-    user_history = UserHistory("Jimbo Wales",
-                               startyear=2022, startmonth=1, startday=1,
-                               endyear=2022, endmonth=1, endday=1)
+    '''Tests user history init'''
+    user_history = UserHistory("QuicoleJR")
 
-    assert user_history.user == "Jimbo Wales"
+    assert user_history.user == "QuicoleJR"
+    assert user_history.tags is None
+    assert user_history.keyword is None
+    assert len(user_history.revisions) > 0
 
 def test_userhistory_tag_filter():
-    """Tests user history filtering"""
-    user_history = UserHistory("Jimbo Wales",
-                               startyear=2021, startmonth=5,
-                               endyear=2021, endmonth=7,
-                               tags=["mw-reverted"])
-
-    assert len(user_history.revisions) == 3
+    '''Tests user history filtering'''
+    user_history = UserHistory("QuicoleJR", tags=["mobile edit"])
+    assert user_history.user == "QuicoleJR"
+    assert len(user_history.tags) == 1
+    assert user_history.tags[0] == "mobile edit"
+    assert len(user_history.revisions) > 0
+    assert "mobile edit" in user_history.revisions[0].tags
 
 def test_userhistory_keyword_filter():
-    """Tests user history filtering"""
-    user_history = UserHistory("Jimbo Wales", keyword="radio",
-                               startyear=2020, startmonth=4,
-                               endyear=2020, endmonth=5)
+    '''Tests user history filtering'''
+    user_history = UserHistory("QuicoleJR", keyword="milk")
+
+    assert user_history.user == "QuicoleJR"
+    assert user_history.keyword == "milk"
+    assert len(user_history.revisions) > 0
+    assert user_history.revisions[0].contains_keyword("milk")
+
+def test_userhistory_timestamps():
+    '''Tests user history with timestamps'''
+    user_history = UserHistory(user="Greatgiant19", startyear=2023, startmonth=2, startday=1,
+                          endyear=2023, endmonth=2, endday=28)
+
+    assert len(user_history.revisions) > 0
+    assert user_history.user == "Greatgiant19"
+    assert user_history.revisions[0].user == "Greatgiant19"
+    assert user_history.revisions[0].title == "Bernie Ecclestone"
```

Comment:
Even if this type of date/time specification ends up in the other tests, I appreciate that it's covered for its own sake with this test.

Reply:
Sounds good, leaving this test and keeping timestamps consistent with other tests